hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -364,8 +364,8 @@ public class HoodieWriteConfig extends HoodieConfig {
 
   public static final ConfigProperty<Boolean> REFRESH_TIMELINE_SERVER_BASED_ON_LATEST_COMMIT = ConfigProperty
       .key("hoodie.refresh.timeline.server.based.on.latest.commit")
-      .defaultValue(false)
-      .withDocumentation("Refresh timeline in timeline server based on latest commit apart from timeline hash difference. By default (false), ");
+      .defaultValue(true)
```

Comment:
I don't know if we are ready for this flip yet. there are probably some open JIRAs

Reply:
This removed in https://github.com/apache/hudi/pull/6179
but @danny0405 why can't we keep it as false? removing it completely is a breaking change. and normally we would at least deprecate it first.

Reply:
> hy can't we keep it as false

It is actually hard code to be true before this patch and this config option is fixed to be effective only in this PR.